scene/gui/graph_edit.h

```diff
@@ -184,7 +184,9 @@ class GraphEdit : public Control {
 	GraphEditMinimap *minimap;
 	void _top_layer_input(const Ref<InputEvent> &p_ev);
 
-	bool is_in_hot_zone(const Vector2 &pos, const Vector2 &p_mouse_pos);
+	bool is_in_input_hotzone(GraphNode gn, int i, const Vector2 &p_mouse_pos)
+	bool is_in_output_hotzone(GraphNode gn, int i, const Vector2 &p_mouse_pos)
```

Comment:
Those two lines are missing semicolons, hence the build failing on every platform. Make sure you test your changes locally, that they compile and that they work and don't cause obvious side-effects.